lib/tbot/service_spiffe_svid_output.go

```diff
@@ -169,17 +254,23 @@ func (s *SPIFFESVIDOutputService) generate(ctx context.Context) error {
 		return trace.Wrap(err, "writing svid certificate")
 	}
 
-	trustBundleBytes := &bytes.Buffer{}
-	for _, ca := range spiffeCAs {
-		for _, cert := range services.GetTLSCerts(ca) {
-			// Values are already PEM encoded, so we just append to the buffer
-			if _, err := trustBundleBytes.Write(cert); err != nil {
-				return trace.Wrap(err, "writing trust bundle to buffer")
+	trustBundleBytes, err := bundleSet.Local.X509Bundle().Marshal()
```

Comment:
I'm having trouble understanding this piece. Here, we are marshalling the federated bundles from `bundleSet.Federated`, as opposed to "whatever was returned from `GetCertAuthorities`. Can you elaborate a bit about what are both of these things and why the result of `GetCertAuthorities` and `services.GetTLSCerts` can now be replaced by this piece of code?

Reply:
Sure. So a "Trust Bundle" is really just the SPIFFE terminology for the information you need to verify that a certificate has been issued by a Certificate Authority. The "local" trust bundle is actually just the SPIFFE CA from your local Teleport cluster. The major change here is that the SPIFFE trust bundle cache introduced in my last PR is now responsible for converting the CertificateAuthority into the TrustBundle format. This lets the conversion be shared by a few different consumers now instead of each consumer independently converting the SPIFFE CertificateAuthority into a trust bundle.